index.html

```diff
@@ -1,62 +1,90 @@
-                  <img
-                     class="logo"
-                     src="IMAGES/logo-HYF.svg"
-                     alt="HackYourFuture logo"
-                  />
-               </a>
-            </div>
-            <!--Navigation section of the header: contains related links-->
+    <!--Adding Google Fonts-->
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Space+Mono:wght@400;700&family=Work+Sans:wght@300;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <!--Adding External Style Sheet-->
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Hack Your Future Belgium</title>
+  </head>
+  <body>
+    <!--Header-->
+    <header>
+      <div class="header-container">
+        <!--Logo-->
+        <div class="hyf-logo">
+          <a href="index.html">
+            <img
+              class="logo"
+              src="IMAGES/logo-HYF.svg"
+              alt="HackYourFuture logo"
+            />
+          </a>
+        </div>
+        <!--Navigation section of the header: contains related links-->
 
-            <div class="nav-menu">
-               <nav>
-                  <ul class="links">
-                     <li><a href="program.html">program</a></li>
-                     <li><a href="#"></a>about</li>
-                     <li><a href="#"></a>volunteer</li>
-                     <li><a href="#"></a>support us</li>
-                     <li><a href="#"></a>faq</li>
-                  </ul>
-               </nav>
+        <div class="nav-menu">
+          <nav>
+            <ul class="links">
+              <li><a href="program.html">program</a></li>
+              <li><a href="#"></a>about</li>
+              <li><a href="#"></a>volunteer</li>
+              <li><a href="#"></a>support us</li>
+              <li><a href="#"></a>faq</li>
+            </ul>
+          </nav>
 
-               <!--Button Apply Now-->
-               <div class="btn-apply">
-                  <a
-                     href="https://docs.google.com/forms/d/e/1FAIpQLSfS0215I6BCy1jzpWmxv9UxYxp5I25ZcoWhSxijhcUImLELcg/viewform"
-                     target="_blank"
-                     class="apply-button"
-                     >apply now</a
-                  >
-               </div>
+          <!--Button Apply Now-->
+          <div class="btn-apply">
+            <a
+              href="https://docs.google.com/forms/d/e/1FAIpQLSfS0215I6BCy1jzpWmxv9UxYxp5I25ZcoWhSxijhcUImLELcg/viewform"
+              target="_blank"
+              class="apply-button"
+              >apply now</a
+            >
+          </div>
+        </div>
+      </div>
+    </header>
+    <!--Main-->
+    <main>
+      <!--Our program-->
+      <section class="our-program">
+        <div class="op-container-2">
+          <div class="op-container-3">
+            <!-- Left Container -->
+            <div class="our-program-left">
+              <h3>Our program</h3>
```

Comment:
I marked the text of the same size as h2. I suppose there is only h1 above, which is the "Helping new talent ..." text. h3 is for example "community' in OUR CORE VALUES section.

Reply:
Yes. It is an h2. I made changes for homepage but not for this one. I will do it.